src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -29,10 +29,14 @@
 
 /**
  * <p>
- * Checks that method and {@code catch} parameter names conform to a format specified
+ * Checks that method parameter names conform to a format specified
```

Comment:
why are we removing catch from description? it still validates `catch` parameter names, just not by default.

Reply:
Hm, https://checkstyle.org/config_naming.html#ParameterName

Do you know test case when it do validation on catch parameter ? Look at PR description, I failed to validate it.

Reply:
Addition of `catch` in javadoc: https://github.com/checkstyle/checkstyle/commit/ac8daf291b44166b386f97f176dd9c5820fabcdd
Split, removing catch from check and making it solo check: https://github.com/checkstyle/checkstyle/issues/2616/

You are correct. I assume javadoc was written correctly.